DI-236 remove stripchr

sophie22 opened the pull request:
remove steps related to removing 'chr' characters from chromosome field in all input files:
* remove unzipping, rezipping and indexing of input VCF files
* remove using samtools to index reference genome file, index file is now a provided input
* which removes the need to install package dependencies
* which removes the need for htslib as an asset dependency

Example jobs that worked:
https://platform.dnanexus.com/projects/GQf1Y704j4jPpxxzqYQYPB2Y/monitor/job/GV06V904j4j1G1k3J9GKX47p

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/eastgenomics/eggd_vcfeval_hap.py/22)
<!-- Reviewable:end -->

sophie22 committed:
added ref index as input, improved wording

sophie22 committed:
correct typo in "stratification"

sophie22 committed:
remove strip chr commands

sophie22 committed:
samtools is no longer required

sophie22 committed:
unpack reference genome file

jethror1 reviewed (changes requested):
> Reviewed 3 of 4 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_vcfeval_hap.py/22)* status: 3 of 4 files reviewed, 1 unresolved discussion (waiting on @sophie22)

___
*[`dxapp.json` line 159 at r1](https://reviewable.io/reviews/eastgenomics/eggd_vcfeval_hap.py/22#-NTTAliTDs5OXEt0i0Df:-NTTAliTDs5OXEt0i0Dg:bsw9u5r) ([raw file](https://github.com/eastgenomics/eggd_vcfeval_hap.py/blob/daaa6647600a233f829baf071d53614e7eafe619/dxapp.json#L159)):*
> ```JSON
>       "systemRequirements": {
>         "*": {
>           "instanceType": "mem1_ssd1_v2_x2"
> ```

do we want to change this default to be bigger if it gives much speed?



_Code quote:_
```JSON
instanceType": "mem1_ssd1_v2_x2"
```


<!-- Sent from Reviewable.io -->

sophie22 committed:
increase instance type for improved performance

https://platform.dnanexus.com/projects/GQf1Y704j4jPpxxzqYQYPB2Y/monitor/job/GV06y284j4jB217G8045vg5z
shaves off 20 minutes runtime and costs half as much

sophie22 reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_vcfeval_hap.py/22)* status: 3 of 4 files reviewed, 1 unresolved discussion (waiting on @jethror1)

___
*[`dxapp.json` line 159 at r1](https://reviewable.io/reviews/eastgenomics/eggd_vcfeval_hap.py/22#-NTTAliTDs5OXEt0i0Df:-NTTC34p94mQ1r1OEC5M:b-zehw8s) ([raw file](https://github.com/eastgenomics/eggd_vcfeval_hap.py/blob/daaa6647600a233f829baf071d53614e7eafe619/dxapp.json#L159)):*
<details><summary><i>Previously, jethror1 (Jethro Rainford) wrote…</i></summary><blockquote>

do we want to change this default to be bigger if it gives much speed?
</blockquote></details>

yes, good idea


<!-- Sent from Reviewable.io -->

jethror1 reviewed (approved):
> Reviewed 1 of 4 files at r1.
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/eggd_vcfeval_hap.py/22)* status: 3 of 4 files reviewed, all discussions resolved (waiting on @sophie22)



<!-- Sent from Reviewable.io -->
